src/pages/Main/MainTop.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import Slider from 'react-slick';
+import styled from 'styled-components';
+
+const MainTop = () => {
+  const [images, setImages] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/mainTopData.json')
+      .then(res => res.json())
+      .then(setImages);
+  }, []);
+
+  const settings = {
+    infinite: true,
+    speed: 500,
+    slidesToShow: 1,
+    autoplay: true,
+    autoplaySpeed: 2000,
+  };
+
+  return (
+    <Slider {...settings}>
+      {images.map(list => (
+        <TopBox key={list.id}>
+          <TopImages src={list.image} />
+
+          <MainLink to="/">{list.name}</MainLink>
+        </TopBox>
+      ))}
+    </Slider>
```

Comment:
키가 많지 않을경우에는 바로 구조분해할당 해줄 수도 있습니다.
참고만 해주세요~

```suggestion
    <Slider {...settings}>
      {images.map(({id, image, name}) => (
        <TopBox key={id}>
          <TopImages src={image} />

          <MainLink to="/">{name}</MainLink>
        </TopBox>
      ))}
    </Slider>
```